stories/module-adsense-components.stories.js

```diff
@@ -189,6 +189,32 @@ const topEarningPagesArgs = {
 	limit: 5,
 };
 
+const topEarningPagesAdSenseReport = {
```

Comment:
Maybe add a comment above this and potentially even rename the variable to something that more clearly indicates that this is only used to get the currency to display, not any "actual" numbers.

Reply:
Good idea, I'll do both.